spec/ameba/rule/performance/any_instead_of_empty_spec.cr

```diff
@@ -51,7 +51,7 @@ module Ameba::Rule::Performance
       issue = source.issues.first
 
       issue.rule.should_not be_nil
-      issue.location.to_s.should eq "source.cr:1:1"
+      issue.location.to_s.should eq "source.cr:1:11"
       issue.end_location.to_s.should eq "source.cr:1:14"
       issue.message.should eq "Use `!{...}.empty?` instead of `{...}.any?`"
```

Comment:
this test can be removed or should be re-worked to the `# ^^^^` notation.

Reply:
Nah, I wouldn't touch it since it's the `reports rule, pos and message` test found all over specs, so there's no point in changing just this one without touching all of the others.